build error with musl libc on alpine Linux

Probably a missing package dependency,  or glibc-specific stuff, there is no `/usr/include/linux/fs.h`:
```
#=== ERROR while installing mirage-block-unix.2.2.0 ===========================#
# opam-version         1.2.2
# os                   linux
# command              ocaml setup.ml -build
# path                 /home/opam/.opam/system/build/mirage-block-unix.2.2.0
# compiler             system (4.02.3)
# exit-code            1
# env-file             /home/opam/.opam/system/build/mirage-block-unix.2.2.0/mirage-block-unix-91-abfa5b.env
# stdout-file          /home/opam/.opam/system/build/mirage-block-unix.2.2.0/mirage-block-unix-91-abfa5b.out
# stderr-file          /home/opam/.opam/system/build/mirage-block-unix.2.2.0/mirage-block-unix-91-abfa5b.err
### stdout ###
# /usr/bin/ocamlopt.opt unix.cmxa -I /usr/lib/ocaml/ocamlbuild /usr/lib/ocaml/ocamlbuild/ocamlbuildlib.cmxa myocamlbuild.ml /usr/lib/ocaml/ocamlbuild/ocamlbuild.cmx -o myocamlbuild
# /home/opam/.opam/system/bin/ocamlfind ocamlc -g -c lib/odirect_stubs.c
# mv odirect_stubs.o lib/odirect_stubs.o
# /home/opam/.opam/system/bin/ocamlfind ocamlc -g -c lib/blkgetsize_stubs.c
# + /home/opam/.opam/system/bin/ocamlfind ocamlc -g -c lib/blkgetsize_stubs.c
# lib/blkgetsize_stubs.c:36:22: fatal error: linux/fs.h: No such file or directory
# compilation terminated.
# Command exited with code 2.
### stderr ###
# E: Failure("Command ''/usr/bin/ocamlbuild' lib/libmirage_block_unix_stubs.a lib/dllmirage_block_unix_stubs.so lib/mirage_block_unix.cma lib/mirage_block_unix.cmxa lib/mirage_block_unix.a lib/mirage_block_unix.cmxs -tag debug' terminated with error code 10")
```

To reproduce:
```
$ git submodule add git://github.com/mirage/mirage-skeleton.git
$ cat >Dockerfile <<EOF
FROM ocaml/opam:alpine
ENV OPAMJOBS 8
RUN opam depext -u -i mirage
WORKDIR /usr/src/app
COPY mirage-skeleton/block/config.ml /usr/src/app/
RUN sudo chown opam -R /usr/src/app && opam config exec -- mirage configure --unix
EOF
$ sudo docker build -t mirage-block-test .
```

--- a/opam
+++ b/opam
@@ -27,3 +27,6 @@ depends: [
   "ocamlbuild" {build}
 ]
 available: [ ocaml-version >= "4.00.0" ]
+depexts: [
+ [["alpine"] ["linux-headers"]]
+]

